pyvista/plotting/qt_plotting.py

```diff
@@ -558,6 +563,17 @@ def window_size(self, window_size):
     def __del__(self):  # pragma: no cover
         self.close()
 
+    def add_callback(self, func, interval=1000, count=None):
```

Comment:
Would you please add a doc string with description of the parameters? Something along the lines of:

```py
 def add_callback(self, func, interval=1000, count=None):
    """Add function that can update the scene in the background

    Parameters
    ----------
    func : callable
        Function to be called with no arguments
    interval : int
        What's this?
    count : int, optional
        What's this?
    """"
    ...
```